pkg/collector/corechecks/snmp/internal/checkconfig/config.go

```diff
@@ -210,11 +211,11 @@ func (c *CheckConfig) GetStaticTags() []string {
 	}
 
 	if c.UseDeviceIDAsHostname {
-		hostname, err := coreutil.GetHostname(context.TODO())
+		hname, err := hostname.Get(context.TODO())
```

Comment:
(BTW, I like `hname` as a variable name here a lot more than `hostnameDetected` which sounds like a boolean "was the hostname detected?" -- but no need to make more changes on that account)

Reply:
You're right, I update the PR since it was already a change